packages/v4/src/content/design-guidelines/components/breadcrumb/breadcrumb.md

```diff
@@ -2,4 +2,21 @@
 id: Breadcrumb
 section: components
 ---
-**Breadcrumbs** act as a resource to help users navigate more efficiently and provide context to help users understand where they are in the application hierarchy. Use breadcrumbs in addition to your global navigation to display a user's location in the application.
+**Breadcrumbs** act as a resource to help users navigate more efficiently and provide context to help users understand where they are in the application hierarchy. 
+
+## Usage
+
+<img src="./img/breadcrumb.png" alt="Example of breadcrumbs" width="990"/>
+
+Use breadcrumbs in addition to your global navigation to display a user's location in the application.
+
+### Guidelines
+* Use breadcrumbs when the primary global navigation items are not always exposed.
+* Never use breadcrumbs as a replacement for global navigation.
+* Never use breadcrumbs to map a user’s personal journey through a website as it may not reflect where the page actually lives. 
```

Comment:
I understand what is being said here, but I might reword. Perhaps say, "Breadcrumbs in PatternFly are intended to show the location of a page in the site hierarchy, and are not intended to map the user's path through the application. "

Reply:
Updated!